engines/duplicity.go

```diff
@@ -208,24 +208,24 @@ func (d *DuplicityEngine) status() (err error) {
 		return
 	}
 
-	fullBackup := fullBackupRx.FindStringSubmatch(stdout)
+	fullBackup := fullBackupRx.FindAllStringSubmatch(stdout, -1)
```

Comment:
There's no need to use `FindAllStringSubmatch` for `fullBackup`. There's only every one occurrence of it.